packages/storybook/stories/buttons.style.scss

```diff
@@ -0,0 +1,26 @@
+[name='Secondary Buttons'] {
+  & button:nth-child(2) {
+    background-color: #dce4ef;
+  }
+  & button:nth-child(3):focus {
```

Comment:
Can you add comments to these describing what's going on, so that people know that the order matters?

Reply:
Added